packages/app-project/src/components/ProjectHeader/helpers/getNavLinks/getNavLinks.js

```diff
@@ -2,9 +2,10 @@ import counterpart from 'counterpart'
 
 function getNavLinks (isLoggedIn, baseUrl, defaultWorkflow) {
   const classifyHref = defaultWorkflow ? `${baseUrl}/classify/workflow/${defaultWorkflow}` : `${baseUrl}/classify`
+  const aboutHref = (process.env.PANOPTES_ENV === 'production') ? `https://www.zooniverse.org${baseUrl}/about` : `${baseUrl}/about/research`
```

Comment:
Staging uses the production API so I think you want to test `APP_ENV` here. At least, I think that's what we called the deployment environment variable.

Reply:
`APP_ENV` is used for sentry: https://github.com/zooniverse/front-end-monorepo#environment-variables

Reply:
I'd like the QA to reflect what we're actually gonna deploy to production for now. When we're ready to deploy the about pages, we can revert it.